### PR TITLE
Remove cg matching rule

### LINKED ISSUE #4: Remove CG matching rule for tAngels
I believe the root is `is_there_edge_between_players`. Look at the methods that this calls. Remove methods that become irrelevant.

### PR DESCRIPTION
**Pull request by seanflyyy**
Made changes to is_there_edge_between_players function to remove all logic pertaining to CG column such that the program still runs as per normal.

### CODE REVIEW

File: .idea/archangel-master.iml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <content url="file://$MODULE_DIR$">
       <excludeFolder url="file://$MODULE_DIR$/venv" />
     </content>
-    <orderEntry type="jdk" jdkName="Python 3.9 (archangel-1)" jdkType="Python SDK" />
+    <orderEntry type="inheritedJdk" />
```
Comment (nicktohzyu) [inline .idea/archangel-master.iml:7]: do not commit this file

File: .idea/misc.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9 (archangel-1)" project-jdk-type="Python SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9 (archangel) (2)" project-jdk-type="Python SDK" />
```
Comment (nicktohzyu) [inline .idea/misc.xml:3]: do not commit this file

File: angel.py [hunk 3]
```diff
@@ -133,14 +133,14 @@ def separate_players(player_list):
...
 '''
-def savegenderlist(genderlist: list):
-    temp = []
-    for k, v in players.items():
-        temp[k] = v.genderplayer
-        temp[k] = v.genderpref
-
-    with open(genderlist.json, 'w+') as f:
-        json.dump(temp, f)
+# def savegenderlist(genderlist: list):
```
Comment (nicktohzyu) [inline angel.py]: Delete unused code please. In the rare case that we want it again in the future, we can always look at the version history, or revert past commits!

File: arrange.py [hunk 4]
```diff
@@ -38,8 +38,6 @@
 # Changing this value changes how much we care about the houses of players being the same
 # If 1 - we don't care, and house de-conflicting is ignored. 0 means we won't allow any players of the same house to be matched.
 
-RELAX_NO_SAME_CG_REQUIREMENT_PERCENTAGE = 0.00
-
 # RELAX_NO_SAME_FACULTY_REQUIREMENT_PERCENTAGE = 0.00 #not used
```
Comment (nicktohzyu) [inline arrange.py]: Delete unused code please.

File: arrange.py [hunk 5]
```diff
@@ -109,26 +99,14 @@ def is_there_edge_between_players(angel_player, mortal_player):
...
-    random_relax_cg_requirement = random.random() < RELAX_NO_SAME_CG_REQUIREMENT_PERCENTAGE
-    if random_relax_cg_requirement:
-        players_are_from_same_cg = False
-    else:
-        players_are_from_same_cg = get_cg_from_player(
-            angel_player) == get_cg_from_player(mortal_player)
-
-
-    valid_pairing = gender_pref_is_respected and (not players_are_from_same_house) and (not players_are_from_same_cg)#and (not players_are_from_same_faculty) # Remove same-house reqr -->  #or players_are_from_same_house) and
+    valid_pairing = gender_pref_is_respected and (not players_are_from_same_house) #and (not players_are_from_same_faculty) # Remove same-house reqr -->  #or players_are_from_same_house) and
```
Comment (nicktohzyu) [inline arrange.py]: Delete unused code please. Don't leave it as a comment. You may use comments when experimenting (when you're unsure of what you might need), but it's not good practise to be kept in production code

**Comment by seanflyyy** [comment]
To test the code, I tried running angel.py after making the necessary changes to the arrange.py and angel.py by deleting the commented out portions and the portions related to CG. 

The program still ran and resulted in an output of a hamilton path.

**Pull request merged**